app/connector/sql/src/test/java/io/syndesis/connector/sql/SqlConnectorIdExistsTest.java

```diff
@@ -0,0 +1,158 @@
+//            return Collections.singletonList("CREATE TABLE ADDRESS (ID INTEGER NOT NULL, "
+//                    + "street VARCHAR(255), number INTEGER, PRIMARY KEY (ID))");
+//        } else {
+//            return Collections.singletonList("CREATE TABLE ADDRESS ("
+//                    + "ID NUMBER GENERATED ALWAYS AS IDENTITY, "
+//                    + "street VARCHAR(255), nummer INTEGER)");
+//        }
+//    }
+
+    @BeforeClass
+    public static void beforeClass() throws SQLException {
+        try (Statement stmt = db.connection.createStatement()) {
+            stmt.executeUpdate("CREATE TABLE ADDRESS (ID INTEGER NOT NULL, "
+                    + "street VARCHAR(255), number INTEGER, PRIMARY KEY (ID))");
+        }
+    }
+
+    @AfterClass
+    public static void afterClass() throws SQLException {
+        try (Statement stmt = db.connection.createStatement()) {
+                stmt.executeUpdate("DROP TABLE ADDRESS");
+        }
+    }
+
+    @Override
+    protected List<Step> createSteps() {
+        return Arrays.asList(
+            newSimpleEndpointStep(
+                "direct",
+                builder -> builder.putConfiguredProperty("name", "start")),
+            newSqlEndpointStep(
+                "sql-connector",
+                builder -> builder.putConfiguredProperty("query", sqlQuery)),
+            newSimpleEndpointStep(
+                "log",
+                builder -> builder.putConfiguredProperty("loggerName", "test"))
+        );
+    }
+
+    // **************************
+    // Parameters
+    // **************************
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        Map<String, Object> parameters = new HashMap<>();
+        parameters.put("number", 14);
+        parameters.put("street", "LaborInVain");
+
+        return Arrays.asList(new Object[][] {
+                { "INSERT INTO ADDRESS (id, street, number) VALUES (1, 'East Davie Street', 100)",
+                        Collections.singletonList(Collections.singletonMap("ID", new String[]{"1"})),
+                        Collections.emptyMap()},
+                { "INSERT INTO ADDRESS (id, street, number) VALUES (1, :#street, :#number)",
+                        Collections.singletonList(Collections.singletonMap("ID", new String[]{"2"})),
+                        parameters}
+        });
+    }
+
+    // **************************
+    // Tests
+    // **************************
+
+    @Test
+    public void sqlConnectorTest() throws Exception {
+        String body;
+        if (parameters.isEmpty()) {
+            body = null;
+        } else {
+            body = JSONBeanUtil.toJSONBean(parameters);
+        }
+
+        
+        try {
+            @SuppressWarnings("unchecked")
+            List<String> jsonBeans = template.requestBody("direct:start", body, List.class);
+        } catch (Throwable t) {
+            t.printStackTrace();
```

Comment:
Is this safe to ignore? Seems it should not be, perhaps the try-catch block needs to be removed

Reply:
added asserts